main.nf

```diff
@@ -1158,7 +1158,7 @@ process split_normalize {
 	script:
 	id = id[0]
 	// rename M to MT because genmod does not recognize M
-	if(params.onco) {
+	if(params.assay != "wgs") {
```

Comment:
Here I think you'll add the "exome" and "myeloid" in addition to the "modycf" in this change. Is this intended? (I.e. before it seems that it was only run for onco, now for 5/6 assays)

Reply:
hmm good catch. I guess exome does not really matter as we never run that, but would change the behavior of myeloid